vars/common.groovy

```
@@ -284,6 +284,9 @@ List<BranchInfo> get_branch_information(Collection<String> tls_branches, Collect
                             deleteDir()
                             checkout_repo.checkout_repo(info)
 
+                            // If a TLS branch doesn't have scripts/abi_check.py, something has gone wrong.
+                            info.has_branch_info = repo == 'tls' || fileExists('scripts/abi_check.py')
```

Why not just check for the file's existence? What do we gain by whitelisting `repo == 'tls'`?

---

Since TLS is always supposed to have `abi_check.py`, I didn't want to silently skip the check in case we screw up the presence check somehow.